integrations/wiz/wiz/client.py

```diff
@@ -1,7 +1,13 @@
+import codecs
+import csv
+import json
+import time
+from contextlib import closing
 from enum import StrEnum
-from typing import Any, AsyncGenerator
+from typing import AsyncGenerator, Dict, List, Any
```

Comment:
Yair encourages us to use the standard `dict` and `list` types instead of Dict and List

Reply:
alright